testing/py3-blis/tox.ini

```diff
@@ -0,0 +1,9 @@
+[tox]
+envlist = py37
+
+[testenv]
+deps = pytest
+    hypothesis
+    numpy
+commands = python setup.py install
+    pytest blis {posargs}
```

Comment:
I don't know Python development well, but doesn't tox run the tests in a virtual env instead of on the host system? That'd make the tests kind of useless for us

Reply:
Also, I don't think you actually use this anymore?

Reply:
I was unable to run tests successfully due to some errors related to python not finding some cython compiled modules. I found an example with `PYTHONPATH` manipulation. Later, at `#alpine-dev`  channel I was provided with example of `tox` use where test run fine, so I extended `tox` method to all packages. I have no sufficient knowledge on how `tox` works. With latest version of this package `PYTHONPATH` trick worked out, but I forgot to remove `tox.ini`, now it is done, thank you!